src/Symfony/Contracts/HttpClient/Test/Fixtures/web/index.php

```diff
@@ -42,6 +42,7 @@
         exit;
 
     case '/head':
+        header('X-Request-Vars: '.json_encode($vars, \JSON_UNESCAPED_SLASHES | \JSON_UNESCAPED_UNICODE));
```

Comment:
This change requires that you also bump the version for `symfony/http-client-contracts` in the dependencies (see the failures of the low deps job).

Reply:
Does this require a new release of `symfony/http-client-contracts` first? If so, should I maybe just find a way to test this without having to change the test server?

Sorry, this is only my second contribution to symfony, so not sure how everything works here.

Reply:
No, you can simply use the version of the patch release to be published next. Our CI will take care of using the code from this PR then.